<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="nzsLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nº of z stacks</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dnaSpin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>53</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="actChk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>actin dna_channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z-pos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dnaChk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>dna dna_channel</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="zSpin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>53</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="renderChk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Render
measurements</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nchLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nº of channels</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="actSpin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>53</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="nfrLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nº of frames</string>
   </property>
  </widget>
  <widget class="QPushButton" name="measureButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Measure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>430</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Condition</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="experimentLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="plotButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
